velox/exec/ContainerRowSerde.cpp

```diff
@@ -587,7 +600,20 @@ int32_t compare(
   using T = typename TypeTraits<Kind>::NativeType;
   T leftValue = left.read<T>();
   T rightValue = right.read<T>();
-  auto result = leftValue == rightValue ? 0 : leftValue < rightValue ? -1 : 1;
+  int32_t result{0};
```

Comment:
This code looks exactly the same as for the other function. Can we abstract this in a common function ?